app/src/main/java/com/onboarding/parkingsystemkotlin/database/ParkingDatabase.kt

```diff
@@ -2,6 +2,9 @@ package com.onboarding.parkingsystemkotlin.database
 
 import com.onboarding.parkingsystemkotlin.entity.Reservation
 import com.onboarding.parkingsystemkotlin.utils.ConstantUtils
+import java.util.*
```

Comment:
remove this wildcard pls, add the specific imports

Reply:
removed, used specific import